### PR TITLE
Cleanup psalm issues in DB/ContactsManager and Console

### PR DESCRIPTION
**Pull request by CarlSchwan**

### CODE REVIEW

File: lib/private/DB/MigrationService.php [hunk 1]
```diff
@@ -342,29 +340,22 @@ public function getMigration($alias) {
...
-	 * @return null|string
-	 */
-	private function getRelativeVersion($version, $delta) {
+	private function getRelativeVersion(string $version, int $delta): ?string {
 		$this->ensureMigrationsAreLoaded();
 
 		$versions = $this->getAvailableVersions();
-		array_unshift($versions, 0);
+		array_unshift($versions, '0');
+		/** @var int $offset */
```
Comment (come-nc) [inline lib/private/DB/MigrationService.php:348]: Why this `@var`? Doesn’t that contradict the comparison with false below?

**Review by come-nc — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Comment by szaimen** [comment]
are test failure related?

**Comment by szaimen** [comment]
phpunit failures...

**Comment by come-nc** [comment]
Rebased on master but tests are failing, not clear why. Something in the changes is breaking DB somehow.

**Pull request merged**